cmd/controller/main.go

```diff
@@ -102,17 +116,84 @@ func main() {
 		log.Fatal(http.ListenAndServe(":"+port, mux))
 	}()
 
+	// initialize opentelemetry
```

Comment:
nit: we could probably extract this in a function to return the object we'll pass to the controller as well as the `defer` function for "cleanly shutdown".

Reply:
I was thinking something like :

```go
trPipelineRun, trTaskRun, deferFn := initializeTracers(…)
defer deferFn()

func initializeTracers(…) (…) {
    # …
    return trPipelineRun, trTaskRun, func() { 
        // the content of the defer below
    }
}
```

And do all "tracer" related code in there. But it's a nit, I'm happy to keep it as is as well 😉

Reply:
It sounds like something we could fix in a follow-up?